finance/funding_requests.md

```diff
@@ -1,82 +1,68 @@
 
-6. At least one member of the finance committee provides approval for the payment.
+Flights should be booked in economy class. If the flight is over 4 hours and the additional cost is less than 20%, a higher class of seating may be booked. We encourage travel plans to be optimized for efficiency. If additional costs are required, a justification should be submitted for approval. Medical reasons are an acceptable justification for higher costs.   
 
-7. Payment is made by NumFocus, typically within 30 days.
+In general, you should attempt to minimize the environmental impact
+of your travel. 
 
+The Finance Committee will aim to provide approval within 1 week of the request.
 
-## Contractor Work
+### After Travel
 
-This document is a first iteration - the interim finance committee and the community as a whole are engaging in a discussion how to develop this process further.
- 
-In its current form, this document describes how the process has been handled in the past, when funding was available only through specific grants for very well defined tasks. So this process will be used initially for existing contributors who require support for priorities already identified by the Astropy coordination committee (as charged by the participants of the 2019 coordination meeting).
+Wthin two weeks of completing your trip you should submit your reimbursment request via the [NumFOCUS Rocket form | https://numfocus.typeform.com/to/oTBLP2] You will need to include itemized receipts for all travel costst. You will also need to provide information for your prefered payment method i.e. back account and routing number, paypal account, etc. 
 
-The intent in the future is to build this into a process that allows both existing and potential new contributors to propose ideas for funding by the Astropy community.  But this requires balancing different priorities for limited funding.  To give time to develop a process for that, this portion of the document is not currently intended for unsolicited proposals, but that is an eventual goal as this process is developed further.
+You should notify the Finance Committee that you have input a request via email so that the payment can be approved. A member of the Finance Committee will review the request and provide approval to NumFOCUS for the payment.
 
-1.  A request for contract work should be made via email to finance@astropy.org.
+Payment will be made to you directly from  NumFocus, typically within 30 days after approval.
 
-2.  If the request is not coming from or already solicited by the coordination committee, the finance committee will seek approval for this contract from the Coordination Committee   
+## Contract Work
 
-3.  The terms of the contract work are negotiated.
+*This document is a first iteration - the Finance Committee and the community as a whole are engaging in a discussion how to develop this process further.*
```

Comment:
Maybe move this to the top of the *document*? It really applies to the whole thing (as opposed to the next line, which is definitely specific to contract work)?